### ISSUE TITLE
Do not include junk like .DS_Store in resources

### ISSUE THREAD

**Issue by rishabhg** [prezi/spaghetti#73]

**Comment by knuton** [comment]
This can be closed, right @rishabhg?

**Comment by lptr** [comment]
Why should this be closed?

**Comment by knuton** [comment]
Ah, then I misunderstood. In this case, the issue should not be closed but contain more explanation. ;)

**Comment by lptr** [comment]
True. IIRC, the problem here is that when you add resources to a module bundle, the Gradle tasks scoop up everything indiscriminately. Most tools try to be clever and ignore hidden files like `.DS_Store` on a Mac, or `.svn` directories etc. Unless you explicitly ask them to be added, that is.
